[2023-03-23 13:52:36,884] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.choose_best manual__2023-03-23T13:52:28.662877+00:00 [queued]>
[2023-03-23 13:52:36,895] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.choose_best manual__2023-03-23T13:52:28.662877+00:00 [queued]>
[2023-03-23 13:52:36,896] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 13:52:36,897] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-03-23 13:52:36,897] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 13:52:36,913] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): choose_best> on 2023-03-23 13:52:28.662877+00:00
[2023-03-23 13:52:36,918] {standard_task_runner.py:52} INFO - Started process 9009 to run task
[2023-03-23 13:52:36,922] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_attack_dag', 'choose_best', 'manual__2023-03-23T13:52:28.662877+00:00', '--job-id', '2723', '--raw', '--subdir', 'DAGS_FOLDER/dynamic_dag.py', '--cfg-path', '/tmp/tmpcumkw84j', '--error-file', '/tmp/tmp1opbxa4u']
[2023-03-23 13:52:36,923] {standard_task_runner.py:80} INFO - Job 2723: Subtask choose_best
[2023-03-23 13:52:36,978] {task_command.py:369} INFO - Running <TaskInstance: multi_attack_dag.choose_best manual__2023-03-23T13:52:28.662877+00:00 [running]> on host 6447f1dacd32
[2023-03-23 13:52:37,042] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_attack_dag
AIRFLOW_CTX_TASK_ID=choose_best
AIRFLOW_CTX_EXECUTION_DATE=2023-03-23T13:52:28.662877+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-23T13:52:28.662877+00:00
[2023-03-23 13:52:37,081] {transport.py:157} INFO - Attempting refresh to obtain initial access_token
[2023-03-23 13:52:37,084] {client.py:777} INFO - Refreshing access_token
[2023-03-23 13:52:38,171] {python.py:173} INFO - Done. Returned value was: None
[2023-03-23 13:52:38,185] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_attack_dag, task_id=choose_best, execution_date=20230323T135228, start_date=20230323T135236, end_date=20230323T135238
[2023-03-23 13:52:38,224] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-23 13:52:38,252] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
